src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Discovery/IPluginInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Discovery
+{
+    /// <summary>
+    ///     A common set of plugin information shared by available plugins and installed plugins.
+    /// </summary>
+    public interface IPluginInfo
```

Comment:
as noted, I would prefer this just be a sealed class

Reply:
Updated to a sealed class after chatted offline.